renovate.json5

```diff
@@ -31,6 +31,11 @@
       matchManagers: ['npm'],
       prPriority: 2,
     },
+    {
+      // Allow renovate to update node
+      matchDepTypes: ['engines'],
+      enabled: true,
```

Comment:
should we add a priority higher than `npm` dependencies? Otherwise this will be way down the list and we'll never get updates until all the npm dependencies are updated, which could be a while

Reply:
oh good idea, I've set it to priority 3, what do you think?